app/ts/background/metadataUtils.ts

```diff
@@ -103,11 +103,11 @@ async function identifyAddressWithoutNode(address: bigint, rpcEntry: RpcNetwork
 	return undefined
 }
 
-export async function identifyAddress(ethereumClientService: EthereumClientService, address: bigint, useLocalStorage = true) : Promise<AddressBookEntry> {
+export async function identifyAddress(ethereumClientService: EthereumClientService, requestAbortController: AbortController | undefined, address: bigint, useLocalStorage = true) : Promise<AddressBookEntry> {
```

Comment:
`AbortController` can be moved as last parameter since it's optional anyway. it would required a lot of changes outside this file tho so can't do suggestions

Reply:
I feel its clear that its always after EthereumClientService in every function that has it